### PR TITLE
Enhancements to Account Management and Introducing Image Handling

### PR DESCRIPTION
**Pull request by Dragonfly911117**
# Title
Enhancements to Account Management and Introducing Image Handling

# Description
This pull request introduces significant enhancements to account management, including new models and improved CRUD operations, along with the adoption of new functionality for image uploads and retrievals. Here's a breakdown of the changes introduced:

1. **Account Management**
   Refined the CRUD operations for accounts (`router/account.py`). The changes in `model/account.py` include new models for handling account data with additional fields.

2. **Image Handling**
   A new feature has been introduced to handle image uploads and retrievals. New routes have been introduced in `router/image.py` and the corresponding models in `model/image.py`. A helper file, `utils/image_io.py` has been added to handle the related operations.

3. **Database Processes**
   `utils/db_process.py` has been modified and includes a new function to check if a record exists in a given database table.

4. **Shop Module Enhancement**
   Minor improvements and cleanups have been made to the Shop router (`router/shop.py`).

5. **Dependencies**
   The `requirements.txt` file has been updated to list specific versions of the dependencies to ensure consistent behavior across different environments.

6. **Server Setup**
   Changes in `server.py` reflect the updates in routing rules, factoring in the new routes introduced in this PR.

Please note: It is essential to thoroughly test these changes in a staging or development environment before rolling out to production. These changes are interdependent and should be deployed together.

Any feedback or questions are appreciated as we aim to continually improve our codebase.

### CODE REVIEW

**Comment by Dragonfly911117** [comment]
> To unzip files at the client-side with pure JavaScript, you can use JSZip, a javascript library for creating, reading, and editing .zip files. Here is an example of how you could use it to download and extract your files:
> Firstly, include JSZip and FileSaver in your project:
> ```html
> <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.5/jszip.min.js"></script>
> <script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/2.0.2/FileSaver.min.js"></script>
> ```
> Then, use the following script for unzipping your file:
> ```js
> let url = "URL_TO_YOUR_ZIP_FILE";  // URL to your zipped file
> 
> fetch(url)
>     .then(response => response.blob())
>     .then(blob => {
>         JSZip.loadAsync(blob).then((zipFiles) => {
>             zipFiles.forEach((relativePath, zipEntry) => {
>                 zipFiles.file(zipEntry.name).async('blob').then((fileData) => {
>                     // These are the files you got.
>                     var blob = new Blob([fileData]);
>                     var link = document.createElement('a');
>                     link.href = window.URL.createObjectURL(blob);
>                     link.download = zipEntry.name;
>                     link.click(); 
>                 });
>             });
>         });
>     });
> ```
> 
> In this example:
> A fetch request is made to your backend's API endpoint (url) that provides the .zip file
> The response.blob() creates a Blob object representing the zipped files.
> We are using JSZip's loadAsync method to read the .zip file and create a zipFiles object.
> Then a loop is created with zipFiles.forEach method to extract each file found in the .zip file asynchronously.
> The unziped file is then converted to a blob and a link is created temporarily for each file in order to start the download. After the file download has been started, the temporary link is discarded.
> link.click() initiates the download of the file.
> This will download each file extracted from the zip file to your default download folder.
> This is a brief overview of the process and might need some adjustment depending on your needs.

--Jetbrains AI

File: router/account.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 )
 
 @router.post(
-    "/create", tags=["create"], responses={
+    "/", tags=["create", "HTTP_POST"], responses={
```
Comment (Vghxv) [inline router/account.py]: tag "create" and "HTTP_POST" is the same concept, remove one of them so swagger looks cleaner, the same thing applies to get/HTTP_GET and update/HTTP_PUT

File: router/account.py [hunk 2]
```diff
@@ -1,125 +1,140 @@
...
+        status.HTTP_404_NOT_FOUND: {
+            "model": ErrorModel
+        }
     }
-})
-async def create_account(
-    account_form: CreateAccountForm = Depends(CreateAccountForm.as_form)
+)
+async def update_account(
+        account_form: UpdateAccountForm = Depends(UpdateAccountForm.as_form)
```
Comment (Vghxv) [inline router/account.py:66]: sussy spaces, check all routers input

**Review by Vghxv — CHANGES_REQUESTED** [review]

File: router/account.py (hunk 1 above)
Comment (PEKKA1117) [reply router/account.py]: fixed with 973f9d6

**Review by Vghxv — APPROVED** [review]

**Review by kyynk — APPROVED** [review]

**Pull request merged**